geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -462,5 +503,19 @@ public int compareTo(OrderedSetEntry o) {
       this.scoreBytes = score;
       this.score = processByteArrayAsDouble(score);
     }
+
+    // Dummy entry used to find the rank of an element with the given score for inclusive or
+    // exclusive ranges
+    OrderedSetEntry(double score, boolean isExclusive, boolean isMinimum) {
+      if (isExclusive && isMinimum || !isExclusive && !isMinimum) {
+        // For use in (this < other) and (this >= other) comparisons
+        this.member = bGREATEST_MEMBER_NAME;
+      } else {
+        // For use in (this <= other) and (this > other) comparisons
+        this.member = bLEAST_MEMBER_NAME;
+      }
+      this.scoreBytes = new byte[] {};
+      this.score = score;
```

Comment:
Do we need to allocate an empty array for scoreBytes? Maybe I'm missing a case, but if this doesn't get used could we just set it to null?

Reply:
Good call. Updated.